### PR TITLE
0.2.0

### PR DESCRIPTION
**Pull request by yeraydiazdiaz**
I thought it'd be good to land the recent changes to minimize the damage of the breaking API changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
 # Changelog
 
-## Unreleased
+## 0.2.0 (2020-01-28)
```
Comment (sethmlarson) [inline CHANGELOG.md]: Totally agree we should be using ISO for the dates.

Comment (yeraydiazdiaz) [reply CHANGELOG.md]: It's actually in https://keepachangelog.com/en/1.0.0/ I missed it for some reason 😄

**Comment by sethmlarson** [comment]
Released to PyPI: https://pypi.org/project/socksio/0.2.0/

**Comment by yeraydiazdiaz** [comment]
Awesome! 🚀

**Pull request merged**